daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -39,63 +43,65 @@ The diagram below is an overview of how Dapr's service invocation works.
 ## Features
 Service invocation provides several features to make it easy for you to call methods between applications.
 
-### Namespace scoping
-
-Applications can be scoped to namespaces for deployment and security, and you can call between services deployed to different namespaces. For more information, read the [Service invocation across namespaces]({{< ref "service-invocation-namespaces.md" >}}) article.
+### Proxying
+- **HTTP proxying**: If you're already using HTTP client libraries within your application, using the HTTP header might be the easiest way to get started. You don't need to change your existing endpoint URLs; just add the Dapr application ID in the HTTP header calls and you're ready to go. For more information, see [Invoke Services using HTTP]({{< ref howto-invoke-discover-services.md >}}). 
+- **gRPC Proxying**: Dapr allows users to keep their own proto services and work natively with gRPC. This means that you can use service invocation to call your existing gRPC apps without having to include any Dapr SDKs or include custom gRPC services. For more information, see the [how-to tutorial for Dapr and gRPC]({{< ref howto-invoke-services-grpc.md >}}).
 
 ### Service-to-service security
 
-All calls between Dapr applications can be made secure with mutual (mTLS) authentication on hosted platforms, including automatic certificate rollover, via the Dapr Sentry service.
+With the Dapr Sentry service, all calls between Dapr applications can be made secure with mutual (mTLS) authentication on hosted platforms, including automatic certificate rollover.
 
 For more information read the [service-to-service security]({{< ref "security-concept.md#sidecar-to-sidecar-communication" >}}) article.
 
-### Access control
-
-Applications can control which other applications are allowed to call them and what they are authorized to do via access policies. This enables you to restrict sensitive applications, that say have personnel information, from being accessed by unauthorized applications, and combined with service-to-service secure communication, provides for soft multi-tenancy deployments.
-
-For more information read the [access control allow lists for service invocation]({{< ref invoke-allowlist.md >}}) article.
-
 ### Retries
```

Comment:
```suggestion
### Resiliency including retries
```

Reply:
This section should say "to find out more read the resiliency article". This one https://docs.dapr.io/operations/resiliency/resiliency-overview/. This section was written before resiliency was added and now needs to be refreshed.

Reply:
Great catch. Added in the link pointing users to the resiliency article.